docs/src/install.rst

```diff
@@ -21,6 +21,14 @@ To install from PyPI you must meet the following requirements:
     MLX is only available on devices running macOS >= 13.3 
     It is highly recommended to use macOS 14 (Sonoma)
 
+
+MLX is also available on conda-forge. To install MLX with conda do:
+
+.. code-block:: shell
+
+   conda install conda-forge::mlx
```

Comment:
Is that the new preferred way of calling it rather than the old `conda install -c conda-forge mlx` ?

Reply:
🤷‍♂️ I assume so as it's written in conda forge : https://anaconda.org/conda-forge/mlx